project_3.html

```diff
@@ -0,0 +1,407 @@
+					<li>Parameters: n (unsigned int)</li>
+					<li>Return: None</li>
+					<li>Description: <code class="pj-code language-arduino">fib()</code> prints the first n elements of the Fibonacci sequence to the console. Each element should be separated by a comma and one space. The last element terminates the line without a space or comma. <code class="pj-code language-arduino">n > 0</code></li>
+					<li>Examples:</li>
+						<ul>
+							<li><code class="pj-code language-arduino">fib(1) : 1</code></li>
+							<li><code class="pj-code language-arduino">fib(4) : 1, 1, 2, 3</code></li>
+						</ul>
+				</ul>
+			</p>
+
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-instructions">Instructions</h2>
+			<h4 class="mb-3" id="pj-heading-checkpoint1">Checkpoint 1: Hello World</h4>
+			<ol class="mb-5 list-group list-group-numbered">
+				<li class="list-group-item">
+					Access the <strong>project template code</strong> through this <a href="https://replit.com/@BenjamenBieleck/uciopsproject3?v=1">link</a>. <strong>Fork the project</strong> to create your own copy in Replit.
+					<p style="text-align:center;">
+						<img class="img-fluid my-3 me-5"
+						src="./assets/images/project_3/fork_label.png">
+					</p>
+				</li>
+				<li class="list-group-item">
+					After you fork the project, the IDE will automatically launch. <strong>Click on the file “ops_project3.cpp”</strong> in the file explorer <strong>(1)</strong>. The file's contents will be displayed in the editor <strong>(2)</strong>. This is where you will implement each of the required functions. When you execute the code, its output will appear in the console <strong>(3)</strong>.
+					<p style="text-align:center;">
+						<img class="img-fluid my-3 me-5"
+						src="./assets/images/project_3/template_code.png">
+					</p>
+				</li>
+				<li class="list-group-item">
+					Now, click on the file “main.cpp” in the file explorer. There, you will see the <code class="pj-code language-arduino">main()</code> function. The provided template code will print “Hello World” to the console.
+				</li>
+				<li class="list-group-item">
+					Click the “Run” button at the top of the viewport. The .cpp file will compile and then execute. You should see the following output to the console:
+					<p style="text-align:center;">
+						<img class="img-fluid my-3 me-5"
+						src="./assets/images/project_3/hello_world_ran.png">
+						<br>
+						“Hello World” is printed onto the console (along with one test for Checkpoint 3).
+					</p>
+				</li>
+			</ol>
+
+			<h4 class="mb-3" id="pj-heading-checkpoint1">Checkpoint 2: Echo</h4>
+			<ol class="mb-5 list-group list-group-numbered">
+				<li class="list-group-item">
+					Implement the <code class="pj-code language-arduino">echo()</code> function as described by the project requirements.
+					<div>
+						<script src="./js/collapsible.js" title="Click to Reveal Hint" content="These objects may be helpful: std::cout, std::cin"></script>
```

Comment:
Please change the text alignment from center to left alignment for all collapsibles here